content/master/concepts/providers.md

```diff
@@ -640,7 +640,7 @@ kind: Provider
 metadata:
   name: provider-gcp-iam
 spec:
-  package: xpkg.upbound.io/upbound/provider-gcp-iam:v0.37.0
+  package: xpkg.upbound.io/upbound/provider-gcp-iam:v1.0.0
```

Comment:
could the `v1` tag just be used here, since this is a manifest that the user would apply?

Reply:
Yup, thanks for catching it.